src/api/types/generated-schema.ts

```diff
@@ -1599,6 +1638,8 @@ export interface components {
       emailNotificationsEnabled: boolean;
       firstName?: string;
       lastName?: string;
+      /** Time zone name in IANA tz database format */
+      timeZone?: string;
```

Comment:
i think we need to update the types under `/src/types` and `src/api/types` etc..

Reply:
Planting Sites and facilities types are defined referencing this file, so no need to update. I just updated Organization and User type